src/main/java/vg/civcraft/mc/civmodcore/api/ItemAPI.java

```diff
@@ -33,10 +33,23 @@ public static boolean isValidItem(ItemStack item) {
 		if (item == null) {
 			return false;
 		}
-		if (item.getType() == Material.AIR) {
+		if (!MaterialAPI.isValidItemMaterial(item.getType())) {
```

Comment:
This is super unneccessary, materials that aren't items will never create ItemStacks, it throws an exception (and the material couldnt be null either)

Reply:
I don't think that's true, or at least not entirely true. I'm looking through Bukkit's code and while there might be an issue if you tried instantiating a new ItemStack with a non-item material, the error doesn't occur when setting the Material of an already created ItemStack. While I agree that the `isValidItem()` is rather over zealous with its checks, it's used in quite a few places now to absolutely ensure that an item is safe to handle.